app/assets/stylesheets/multi-select.scss

```diff
@@ -0,0 +1,10 @@
+// Styles for the MultiSelect.js component
+
+.input-custom-height {
+  // Don't override input height and margin values
+  input {
+    height: initial;
```

Comment:
how does this initial prop work?

Reply:
we spoke on slack about this, it just sets this value to whatever the default is, as if it had never been set.